src/test/java/com/example/jwt/JwtApplicationTests.groovy

```diff
@@ -1,17 +1,21 @@
 package com.example.jwt;
 
-import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest
-import org.springframework.test.web.servlet.MockMvc;
+import spock.lang.Shared;
 import spock.lang.Specification;
 
 @SpringBootTest
 class JwtApplicationTests extends Specification {
 
+    @Shared
+    def client = new RESTClient( "$SERVER_URL:$SERVER_PORT")
```

Comment:
Where is the RESTClient imported from?
and where the $SERVER_URL and $SERVER_PORT come from?